documentation/dev-portal/src/quickstart/overview.md

```diff
@@ -4,8 +4,7 @@ There are multiple options to quickly connect to Nym and see the network in acti
 
 Demo application:
 * a browser-based 'hello world' [chat application](https://chat-demo.nymtech.net). Either open in two browser windows and send messages to yourself, or share with a friend and send messages to each other through the mixnet!
-
-Proxy traffic with the Nym Socks5 client:
+* a Coconut-scheme based [Credential Library](https://coco-demo.nymtech.net/). This is a WASM implementation of our Coconut libraries which generate raw Coconut credentials. Test it to create and re-randomize your own credentials!
```

Comment:
Move this to the section above ( demo applications ) - this doesn't use the socks5 client